exercises/joseph_ramsay/ex02_oo_basics/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required ( VERSION 2.8.5 )
+
+###############################################################################
+# Setup
+###############################################################################
+# Name
+project ( Shapes )
+
+# Find custom cmake modules
+set ( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../CMake")
+
+# Common setup
+include ( CommonSetup )
+
+###############################################################################
+# Executable
+###############################################################################
+
+set ( SRC_FILES
+  src/main.cpp
+  src/rectangle.cpp
+  src/square.cpp
+  src/circle.cpp
+  src/triangle.cpp
+  src/shapeSorter.cpp
+  # add any additional source files here
+)
+
+set ( HDR_FILES
+  # add any additional header files here
```

Comment:
please register the header files in the `CMakeLists.txt` or else it will not show up in Visual Studio